integration_tests/integration/referral.spec.js

```diff
@@ -14,7 +15,13 @@ describe('Referral form', () => {
 
   it('User starts a referral, fills in the form, and submits it', () => {
     const deliusServiceUser = deliusServiceUserFactory.build({ firstName: 'Alex' })
-
+    const randomInterventionId = '99ee16d3-130a-4d8f-97c5-f1a42119a382'
+    const intervention = interventionFactory.build({
+      title: 'Better solutions (anger management)',
+      serviceCategories: ['accommodation'].map(name => serviceCategoryFactory.build({ name })),
```

Comment:
Did you mean to put more in here, or is there a reason you've mapped over the single element array, rather than `serviceCategoryFactory.build({name: accommodation})`?

Reply:
I did it as an array to make it easier to test it with a cohort service behaviour by just plugging in a second string into the array